src/content/developers/docs/consensus-mechanisms/pow/mining/index.md

```diff
@@ -70,7 +76,7 @@ Ethereum Mainnet only ever used one mining algorithm - ['Ethash'](/developers/do
 
 ## Further reading {#further-reading}
 
-- [What does it mean to mine Ethereum?](https://docs.ethhub.io/using-ethereum/mining/) _EthHub_
+- [What does it mean to mine Ethereum?](https://web.archive.org/web/20220917012350/https://docs.ethhub.io/using-ethereum/mining/) _EthHub (Archived)_
```

Comment:
Do we want to be referencing content like this that has been deprecated?

Reply:
I left it in for this specific case as Eth mining has also been "deprecated". As such, unlikely that any new content will be published for this topic

Reply:
This seems like a reasonable rationale to me. Would you have any other PoW suggestions to replace this with @wackerow?